solutions/fully-configurable/main.tf

```diff
@@ -0,0 +1,201 @@
+    },
+    metrics_data = {
+      enabled         = true
+      bucket_crn      = module.buckets.buckets[local.metrics_bucket_name].bucket_crn
+      bucket_endpoint = module.buckets.buckets[local.metrics_bucket_name].s3_endpoint_direct
+    }
+  }
+  logs_routing_tenant_regions   = var.logs_routing_tenant_regions
+  skip_logs_routing_auth_policy = var.skip_logs_routing_auth_policy
+  policies                      = var.logs_policies
+}
+
+#######################################################################################################################
+# COS
+#######################################################################################################################
+
+locals {
+  use_kms_module    = var.kms_encryption_enabled_buckets && var.existing_kms_key_crn == null
+  kms_region        = var.kms_encryption_enabled_buckets ? module.existing_kms_crn_parser[0].region : null
+  existing_kms_guid = var.kms_encryption_enabled_buckets ? module.existing_kms_crn_parser[0].service_instance : var.existing_kms_key_crn != null ? module.existing_kms_key_crn_parser[0].service_instance : null
+  kms_service_name  = var.kms_encryption_enabled_buckets ? module.existing_kms_crn_parser[0].service_name : var.existing_kms_key_crn != null ? module.existing_kms_key_crn_parser[0].service_name : null
+  kms_account_id    = var.kms_encryption_enabled_buckets ? module.existing_kms_crn_parser[0].account_id : var.existing_kms_key_crn != null ? module.existing_kms_key_crn_parser[0].account_id : null
+
+  data_bucket_name    = "${local.prefix}${var.cloud_logs_data_cos_bucket_name}"
+  metrics_bucket_name = "${local.prefix}${var.cloud_logs_metrics_cos_bucket_name}"
+  cos_instance_guid   = module.existing_cos_instance_crn_parser.service_instance
+
+  key_ring_name = "${local.prefix}${var.cloud_logs_cos_key_ring_name}"
+  key_name      = "${local.prefix}${var.cloud_logs_cos_key_name}"
+  kms_key_crn   = var.kms_encryption_enabled_buckets ? var.existing_kms_key_crn != null ? var.existing_kms_key_crn : module.kms[0].keys[format("%s.%s", local.key_ring_name, local.key_name)].crn : null
+  kms_key_id    = var.existing_kms_instance_crn != null ? module.kms[0].keys[format("%s.%s", local.key_ring_name, local.key_name)].key_id : var.existing_kms_key_crn != null ? module.existing_kms_key_crn_parser[0].resource : null
+
+  create_cross_account_auth_policy = var.existing_cloud_logs_crn == null ? !var.skip_cos_kms_iam_auth_policy && var.ibmcloud_kms_api_key == null ? false : true : false
+}
+
+module "existing_cos_instance_crn_parser" {
+  source  = "terraform-ibm-modules/common-utilities/ibm//modules/crn-parser"
+  version = "1.1.0"
+  crn     = var.existing_cos_instance_crn
+}
+
+module "buckets" {
+  providers = {
+    ibm = ibm.cos
+  }
+  depends_on = [time_sleep.wait_for_authorization_policy[0]]
+  source     = "terraform-ibm-modules/cos/ibm//modules/buckets"
+  version    = "8.19.3"
+  bucket_configs = [
```

Comment:
the bucket configs seem to be missing `activity_tracking`, `metrics_monitoring` and `force_delete` options. For reference, see https://github.com/terraform-ibm-modules/terraform-ibm-scc/blob/83983918ab1c4ca857d2a1b4c748b5dfaac85e93/solutions/fully-configurable/main.tf#L151-L162